File: solutions/javascript/two-fer/1/two-fer.js
//
// This is only a SKELETON file for the 'Two fer' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const twoFer = (name) => {
  // If name is not provided, use "you"
  const person = name || 'you';
  return `One for ${person}, one for me.`;
};

File: solutions/python/two-fer/1/two_fer.py
"""Provide the 'two-fer' dialogue."""


def two_fer(name: str = "you") -> str:
    """Return the two-fer dialogue for the given name.

    Args:
        name: The name of the person to give the extra cookie to.
              Defaults to "you".

    Returns:
        A string of the form "One for <name>, one for me."
    """
    return f"One for {name}, one for me."